### DISCUSSION TITLE
Problem with the Simple-Mesh layer

### DISCUSSION THREAD

**Discussion by NadineSarah** [visgl/deck.gl#6268] [Q&A]
Hey :) 
Background: I have rewritten the simple mesh layer to accept an array of data and use it to create a terrain profile in the given area (basically similar to the terrain layer). For this I have also rewritten the terrain loader to use the given array as z coordinates to create the mesh that the simple mesh layer needs. 
The problem now is that when moving the map in y direction (from top to bottom or the other way around) the height of the 'mountains' changes. 
As an example:
1. Bevor moving the map:
![image](https://user-images.githubusercontent.com/39406788/136063273-5266a927-2d0f-46ef-9937-15ec0d0bb083.png)
2. After moving the map in positiv y direction
![image](https://user-images.githubusercontent.com/39406788/136063393-c46b9391-5ccd-4e44-a459-a875f0c2a077.png)

To me it doesn't look like this is a general problem of the simple mesh layer. Does anyone have an idea what this could be or in which area of my code I should look for errors?

**Comment by justdigi** [comment]
I'm curious if you ever got this to work?
